resources/examples/manager/BasicAssetLibrary/plugin/BasicAssetLibrary/BasicAssetLibraryInterface.py

```diff
@@ -110,11 +110,18 @@ def entityExists(self, entityRefs, context, hostSession):
             results.append(result)
         return results
 
-    def resolve(self, entityRefs, traitSet, context, hostSession):
+    def resolve(self, entityRefs, traitSet, context, hostSession, successCb, errorCb):
         if context.isForWrite():
-            return [EntityResolutionError("BAL entities are read-only", ref) for ref in entityRefs]
-        results = []
-        for ref in entityRefs:
+            # TODO(DF): Should this be a whole-batch error (i.e.
+            #  exception) or per-element error?
```

Comment:
[fix] Remove TODO. This is the correct behavior I think, as BAL just happens to have this as a holistic response. For the majority of cases, this _is_ a per-element scenario (ie. in any real-world asset system that supports publishing), so we definitely need consistent behaviour, and exceptions should only really be for high-level exceptional happenings that affects the ability to derive _any_ data (eg: server dead).

Reply:
Good spot. Removed in 74321bbc